recipes/haystack_bio/meta.yaml

```diff
@@ -16,12 +16,11 @@ build:
     - "haystack_tf_activity_plane = haystack.generate_tf_activity_plane:main"
     - "haystack_download_genome = haystack.download_genome:main"
     - "haystack_run_test = haystack.haystack_common:run_testdata"
-  number: 1
+  number: 3
```

Comment:
this should be 2 isn't it?

Reply:
@bgruening I cancelled the second build that is why. Thanks for fixing it!